src/components/project/ProjectsList.tsx

```diff
@@ -72,12 +77,9 @@ const ProjectsList = (props: ProjectsListProps) => {
       <Table size="sm" style={{ marginTop: 10 }}>
         <ProjectsListHeader />
         <tbody>
-          {filteredProjects.length > 0 && filteredProjects.map(project => {
-            if (project.details.isActive === !isShowingInActiveProjects) return (
-              <ProjectsListRow project={project} key={project.details._id} />
-            )
-            return false
-          })}
+          {filteredProjects.length > 0 && filteredProjects.map(project => (
```

Comment:
dit is overbodig, je kan de `filteredProjects.length > 0 && ` gewoon weglaten voor hetzelfde resultaat